controllers/operations/opsrequest_controller.go

```diff
@@ -123,9 +124,16 @@ func (r *OpsRequestReconciler) fetchOpsRequest(reqCtx intctrlutil.RequestCtx, op
 
 // handleDeletion handles the delete event of the OpsRequest.
 func (r *OpsRequestReconciler) handleDeletion(reqCtx intctrlutil.RequestCtx, opsRes *operations.OpsResource) (*ctrl.Result, error) {
-	if opsRes.OpsRequest.Status.Phase == appsv1alpha1.OpsRunningPhase && !opsRes.Cluster.IsDeleting() {
+	if opsRes.OpsRequest.Status.Phase == opsv1alpha1.OpsRunningPhase && !opsRes.Cluster.IsDeleting() {
 		return nil, nil
 	}
+	if opsRes.Cluster.IsDeleting() && opsRes.OpsRequest.DeletionTimestamp.IsZero() {
```

Comment:
What if the cluster has already been deleted (i.e. object not found)? Will the opsReq still be deleted?

Reply:
ops is owned by cluster, ops will be automatically deleted when the cluster itself is deleted. 
but due to tmp pods created by ops may mount instance PVCs, which can prevent PVCs from being released then cluster will be always deleting.

Reply:
Got it, please add a comment about this logic to the code, because it's not so obvious why an explicit deletion is needed.